yang/ietf-bearer-svc.yang

```diff
@@ -111,6 +111,11 @@ module ietf-bearer-svc {
   grouping location-information {
     description
       "Basic location information";
+    leaf locationName {
+        type string;
+        description
+          "Refers to 3GPP NRM {{TS-28.623}} Clause A 2.2.2 IOC ManagedElement. The physical location (e.g., an address) of an (3GPP) entity. It may contain no information to support the case where the derivative of ManagedElement needs to represent a distributed multi-location NE.";
```

Comment:
```suggestion
          "Provides a location name. This data node can be mapped, e.g., to the 3GPP
           NRM IOC ManagedElement.";
```

Reply:
Thank @Ivan-Bykov-RBBN. The description should be generic. It is OK to provide NRM as an example.
Please let me know if the proposed change is OK with you.

Reply:
Agreed.
Thank you!